source/geod24/concurrency.d

```diff
@@ -925,35 +925,45 @@ protected:
     static class InfoFiber : Fiber
     {
         ThreadInfo info;
+        FiberCondition cond;
 
         this(void delegate() op, size_t sz = 512 * 1024) nothrow
         {
+            cond = null;
             super(op, sz);
         }
     }
 
     protected class FiberCondition
     {
+        MonoTime limit = MonoTime.init;
```

Comment:
Could use a ddoc-style comment with `///`.

Reply:
Side-note: A lot of the existing fields are undocumented, they were taken verbatim from the Phobos implementation. But our style is to document all fields. It would be good to at least document the newly added ones and document the other ones later.